geotrek/common/models.py

```diff
@@ -178,7 +178,7 @@ def pictogram_off(self):
 
 
 class RecordSource(OptionalPictogramMixin):
-    name = models.CharField(verbose_name=_("Name"), max_length=50)
+    name = models.CharField(verbose_name=_("Name"), max_length=80)
```

Comment:
Why 80 ?

Reply:
50 is the current max length and it is slightly too short for some names. IMO increasing the length to 80 characters looks acceptable to accommodate APIDAE users.